src/c_api/c_api.cc

```diff
@@ -91,6 +91,12 @@ int MXRandomSeed(int seed) {
   API_END();
 }
 
+void MXInitialise() {
```

Comment:
What happens if this isn't called?

Reply:
I put in the example code above, but the functions are not registered so if you try and load a symbol from a file before calling it you will get an error:

Check failed: op != nullptr Operator FullyConnected is not registered

Reply:
So wouldn't this rather be a task to do when you load a file, rather than having this as a standalone function?

Reply:
If you look at src/c_api/c_predict_api.cc lines 111 
There is a bit where all the symbols are registered with MXListAllOpNames 
I think it would be better to use a function whose name is clear, which can be inserted whenever it is needed such as in file loading and the prediction API.